src/views/Jobs/JobResultsTable.vue

```diff
@@ -1003,11 +1003,32 @@ export default Vue.extend({
                 manipulation => {
                   // First find the original bounds for the reaction, because one of them
                   // will need to be part of the request to modify bounds.
-                  const model_reaction = this.model.model_serialized.reactions.find(
+                  let oldLowerBound;
+                  let oldUpperBound;
+                  // Try to find the reaction among the card's added reactions, in case
```

Comment:
Do we need the same in `PathwayPredictor+CofactorSwap` https://github.com/DD-DeCaF/caffeine-vue/blob/2aef6ab47b5f1f5f9bc8b8b2f27ab4ca83c7442b/src/views/Jobs/JobResultsTable.vue#L1069-L1081

Reply:
I don't think any heterologous reactions would be used for the swap? /cc @Midnighter

Reply:
That *can* actually happen. If any heterologous reaction uses NADH or NADPH and it turns out that in the organism's metabolism the other co-factor performs better then it will become a swap target.

Reply:
Created card: https://app.zenhub.com/workspaces/bioviz-scrum-5d22fe4139b1324e0011234c/issues/dd-decaf/scrum/1027